src/components/Main/validation/schema.ts

```diff
@@ -156,11 +156,15 @@ export const schema: yup.Schema<FormData> = yup
             population: positiveIntegerSchema.required(MSG_REQUIRED),
             confirmed: percentageSchema.required(MSG_REQUIRED),
             severe: percentageSchema.required(MSG_REQUIRED),
+            palliative: percentageSchema.required(MSG_REQUIRED),
             critical: percentageSchema.required(MSG_REQUIRED),
             fatal: percentageSchema.required(MSG_REQUIRED),
             isolated: percentageSchema.required(MSG_REQUIRED),
           })
-          .required(MSG_REQUIRED),
+          .required(MSG_REQUIRED)
+          .test('max', MSG_EXCEED_100, ({ critical, palliative }: SeverityDistributionDatum) => {
+            return critical + palliative <= 100
+          }),
```

Comment:
Moved the form validation test to the object scope, so that it is easier to understand and type properly.